.github/workflows/Tests.yml

```diff
@@ -86,15 +86,24 @@ jobs:
           cd zimui
           $(yarn bin)/cypress run
 
-  build-docker:
+  # this job replaces the standard "build_docker" job since it builds the docker image
+  run-integration-tests:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
 
       - name: Ensure we can build the Docker image
         run: |
-          docker build -t testimage .
+          docker build -t youtube2zim .
 
       - name: Ensure we can start the Docker image
         run: |
-          docker run --rm testimage
+          docker run --rm youtube2zim
+
+      - name: Run scraper
+        env:
+          YOUTUBE_API_KEY: ${{ secrets.YOUTUBE_API_KEY }}
+        run: docker run -v $PWD/output:/output youtube2zim youtube2zim --api-key "$YOUTUBE_API_KEY" --type channel --id "UC8elThf5TGMpQfQc_VE917Q" --name "openZIM_testing" --zim-file "openZIM_testing.zim"
```

Comment:
I think we should also pass an optimization cache URL, otherwise the CI will spend too much time reencoding videos. WDYT? If you agree, I will create a dedicated Wasabi user for this, and we will need a new CI secret

Reply:
Sure sounds like a good idea. I'll update the workflow command.